examples/run/dp1.x-lammps-ABACUS-lcao-dpks/methane/param.json

```diff
@@ -0,0 +1,155 @@
+{
+  "type_map": ["H", "C"],
+  "mass_map": [1.0, 12.0],
+
+  "_comment": "initial data set for Training and the number of frames in each training  batch",
+  "init_data_prefix": "/home/mhchen_pkuhpc/mhchen_cls/lustre2/5_liurenxi/5_ABACUS_dpgen_interface/5_dpgen_examples/dpgen-example/run_abacus/",
+  "init_data_sys":   [
+    "abacus_init_data"
+  ],
+  "init_batch_size": [
+    8
+  ],
+
+  "_comment": "configurations for starting MD in Exploration and batch sizes when traning snapshots derived from these configs (if they were selected)",
+  "sys_configs_prefix": "/home/mhchen_pkuhpc/mhchen_cls/lustre2/5_liurenxi/5_ABACUS_dpgen_interface/5_dpgen_examples/dpgen-example/run",
+  "sys_configs":    [
+    [
+      "CH4.POSCAR.01x01x01/01.scale_pert/sys-0004-0001/scale-1.000/000000/POSCAR",
+      "CH4.POSCAR.01x01x01/01.scale_pert/sys-0004-0001/scale-1.000/000001/POSCAR",
+      "CH4.POSCAR.01x01x01/01.scale_pert/sys-0004-0001/scale-1.000/00000[2-9]/POSCAR"  
+    ],
+    [
+      "CH4.POSCAR.01x01x01/01.scale_pert/sys-0004-0001/scale-1.000/00001*/POSCAR"
+    ]
+  ],
+  "sys_batch_size": [
+    8,
+    8
+  ],
+
+  "_comment": " 00.train ",
+  "numb_models":            4,
+
+  "default_training_param": {
+    "model": {
+      "type_map":           ["H","C"],
+      "descriptor": {        
+        "type":             "se_a",
+        "sel":              [16,4],
+        "rcut_smth":        0.5,
+        "rcut":             5.0,
+        "neuron":           [10,20,40],
+        "resnet_dt":        false,
+        "axis_neuron":      12,
+        "seed":             0
+      },
+      "fitting_net": {
+        "neuron":           [120,120,120],
+        "resnet_dt":        true,
+        "coord_norm": true,
+        "type_fitting_net": false,
+        "seed": 0
+      }
+    },
+      "loss": {
+        "start_pref_e":      0.02,
+        "limit_pref_e":      2,
+        "start_pref_f":      1000,
+        "limit_pref_f":      1,
+        "start_pref_v":      0,
+        "limit_pref_v":      0
+    },
+      "learning_rate": {
+        "type":              "exp",
+        "start_lr":          0.001,
+        "decay_steps":       180,
+        "decay_rate":        0.95
+    },
+      "training": {
+        "systems":           [],
+        "set_prefix":        "set",
+        "stop_batch":        3000,
```

Comment:
It seems this is DeePMD-kit 1.x version's parameters. Could you please update the example for DeePMD-kit 2.x?  See https://github.com/deepmodeling/dpgen/blob/devel/examples/run/dp2.x-lammps-vasp/param_CH4_deepmd-kit-2.0.1.json

Reply:
I have updated the "training" parameters in both this file and the abacus-pw example file.